ArduPlane/ArduPlane.cpp

```diff
@@ -231,15 +231,14 @@ void Plane::update_compass(void)
  */
 void Plane::update_logging1(void)
 {
-    if (should_log(MASK_LOG_ATTITUDE_MED) && !should_log(MASK_LOG_ATTITUDE_FAST)) {
+    // MASK_LOG_ATTITUDE_FULLRATE logs at 400Hz, MASK_LOG_ATTITUDE_FAST at 25Hz, MASK_LOG_ATTIUDE_MED logs at 10Hz
+    // higest rate selected wins
+
+    bool log_fast = (should_log(MASK_LOG_ATTITUDE_FULLRATE) || should_log(MASK_LOG_ATTITUDE_FAST));
+    if (should_log(MASK_LOG_ATTITUDE_MED) && !log_fast) {
         Log_Write_Attitude();
+        ahrs.Write_AOA_SSA();  // AOA only loggged if MASK_LOG_ATTIUDE_MED is active
```

Comment:
so AOA is *never* logged if attitude is fast? That sounds bad.

Reply:
nope...but easy to change if we need it logged at other speeds....currently only if ATTITUDE_MED bit is set...the default logs it...but you could set only FAST and/or FULLRATE and it would not be logged

Reply:
I was about to call that a regression but its actually an existing bug! AOA should get logged at least 10Hz or so

Reply:
I can fix it to do that easily...but need some guidance on what can be logged at 10Hz vs 25hz....lots of things were at 10HZ long ago, evidently, then got a blanket change to 25Hz....fine if this is what we want, I can fix that....then no need for some code in ATTITUDE_FAST...so of the logging1 and logging2 groups, what needs to be what speed?